miniconf-py/setup.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/python3
+"""
+Author: Vertigo Designs, Ryan Summers
+
+Description: Setup file for Miniconf packaging.
+"""
+from setuptools import setup, find_packages
+
+# Load the version string from the version file.
+execfile('miniconf/version.py')
+
+setup(name='miniconf-rs',
```

Comment:
Maybe
```suggestion
setup(name='miniconf-py',
```
?

Reply:
This is the name it gets published on `pypi` as - I'd prefer if it was just `miniconf`, but that's taken, so I was opting for `miniconf-rs` so that users would be able to easily correlate it to the rust `miniconf` package. Open to renaming suggestions though. I think `miniconf-py` is a bit difficult to discern if you're just looking at pypi.

Reply:
Then miniconf-mqtt as this is an impl on one specific transport.